### PR TITLE
PAT 1662: Remove unused RS Classes

### PR DESCRIPTION
**Pull request by Gryzor**
### Objective
* Remove some of the unused classes from the original ResearchStack fork, that neither Axon, Cortex, nor PAT used. 
* Remove unused dependencies from ResearchStack after the classes have been removed. 

### Description
* `StorageAccess` was a Singleton class from ResearchStack (RS), that was designed to be supplied with access to a filesystem, as well as a database-backed storage system for TaskResponses.
* This _singleton_ supported Aes encryption via a PIN code, and a whole lot of other abstractions around the concept of storing a Task Response in a SQLite DB.
* When Medable forked RS, it replaced all that (leaving the classes in place) with a version that did nothing (literally and practically), but left the implementations with emtpy stubs, this pull request's aim is to remove all this.
* After removing all the unused classes, and retrofiting TaskActivity to extend directly from AppCompat, PinCodeActivity became useless. With it, StorageAccess cascaded a deletion of a whole lot of classes that were never used. 
* Last but not least, the gradle dependencies had to go too, this removed some strange annotation processors, that were slowing down the build (Squeaky didn't support "incremental", so it had to be re-compiled every time). 

### Branches
Simply put, checkout `bug/PAT-1662` in RS, Axon, and PAT. (Cortex is `development`).

### Testing?
* There's nothing new here, but if you can complete a task in any study, that's probably a good enough test. QA will eventually perform a Smoke Test regression before the release.

### Ticket(s)?
* PAT-1662

### CODE REVIEW

**Comment by jaimesDSG** [comment]
It's working fine for me.
